### ISSUE TITLE
Missing "build number" and other stuff

### ISSUE THREAD

**Issue by moul** [berty/yolo#251]
![manfreds-imac - Screen Shot 2020-05-18 at 11 20 43](https://user-images.githubusercontent.com/94029/82196135-9dcda480-98f9-11ea-9634-3ff87cc7508b.png)

this build -> https://yolo.berty.io/?build_id=https://buildkite.com/berty/berty/builds/1919

you can see "iOS 1919", but the build line itself has a link to 1919, but not the "1919" text anywhere

---

Also, this build has two builds, 1919 and 634, it's something caused by the way we use buildkite:
1. when someone push, it triggers the "634" build (or the other one, I don't remember)
2. we click on the build and we have a manual button on buildkite to trigger the actual real build which basically is a webhook that generates a safe build

So, basically ->
* https://yolo.berty.io/?build_id=https://buildkite.com/berty/berty/builds/1919
* and https://yolo.berty.io/?build_id=https://buildkite.com/berty/berty-open/builds/634

are related, but I don't know yet "how-to" support them, any idea?

**Comment by moul** [comment]
Ok, I get it

build 634 was the build against the pull, 1919 is the build against master after merging, so basically 634 is the "purple" build of the pull 1945, and we can have multiple builds

---

I suggest you to:
* be sure that 634 is fully displayed on not only the build, we should also see the artifact
* put 634 and its artifacts (and eventual other builds) in a collapsible section (as we already discussed)

**Comment by ekelen** [comment]
Sorry for the eighth re-reading of this. Can you confirm it's correct in current deployment as below (for the collapse/expand behavior as well as the ID numbers displayed)?

<img width="712" alt="image" src="https://user-images.githubusercontent.com/24300177/82617541-aca1a900-9bd0-11ea-8d70-c9d73dc31657.png">

**Comment by moul** [comment]
🕚 Closing as no recent updates. Please reopen if you need any updates to this ticket.
